README.md

```diff
@@ -1,24 +1,20 @@
 # Introduction
 
-This repository houses all the training data , models and code to make said models utilized in the PZH Natura 2000 remote sensing project. The aim of this project is to employ machine learning techniques on land satellite imagery to monitor the distribution of various nature types within protected areas over time, in alignment with diverse climate/nature policies. Therefore, the models herein predict the nature type for each pixel in a land satellite image, which could include Grass, Forest, Sand, etc., based on learned annotations.
+This repository contains all the necessary resources—training data, models, and code—used in the PZH (Natura 2000) remote sensing project on NSO satellites. The goal of this project is to leverage machine learning techniques to analyze satellite imagery, enabling the monitoring of land distribution across various (nature) types within protected Natura 2000 areas over time. Our approach involves training models to classify individual pixels into specific (nature) classes based on their spectral characteristics.
 
-The annotated pixels are stored as pandas dataframes download links in the data/annotations folder and are publicly available for anyone to train their own model. In essence, we used pixel values from the land satellite images to annotate various nature types. Our objective is to predict the type of nature based on these pixel values.
+Through our research, we have determined that supervised machine learning models yield the best results. However, these models require extensive training data, which necessitated the manual annotation of satellite image pixels.
 
-The models themselves can be downloaded with the links found in the readme.md file in the saved_models folder.
+The annotated pixels are organized in pandas DataFrames, which are available for download in the data/annotations folder. These annotations are publicly accessible, allowing others to use them for their own model training. Essentially, we've labeled pixel values in the satellite imagery to correspond with different land types.
```

Comment:
This is not true. The annotated pixels are organised in gkpg files, shape files and geojson files. None of which are pandas DataFrames.

Reply:
I have fixed this too.